pkgs/applications/networking/instant-messengers/ripcord/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib, stdenv, fetchurl, writeScript, appimageTools, buildFHSUserEnv, makeFontsConf,
+  gsettings-desktop-schemas, gtk3, twemoji-color-font }:
+
+let
+  pname = "ripcord";
+  version = "0.4.23";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://cancel.fm/dl/Ripcord-${version}-x86_64.AppImage";
+    sha256 = "0395w0pwr1cz8ichcbyrsscmm2p7srgjk4vkqvqgwyx41prm0x2h";
+    name = "${pname}-${version}.AppImage";
+  };
+
+  fontsConf = makeFontsConf {
+    fontDirectories = [
+      twemoji-color-font
+    ];
+  };
+
+  appimageContents = appimageTools.extractType2 {
+    inherit name src;
+  };
+
+in buildFHSUserEnv (appimageTools.defaultFhsEnvArgs // {
+  inherit name;
+
+  targetPkgs = pkgs: appimageTools.defaultFhsEnvArgs.targetPkgs pkgs ++ (with pkgs; [ ]);
+
+  extraBuildCommands = ''
+    # Begin patch to fix https://dev.cancel.fm/tktview?name=d2dc78360c
+    chmod u+w ./etc
+
+    # Realise fonts by one level
+    rm ./etc/fonts && mkdir ./etc/fonts
```

Comment:
at a glance I don't see a `/etc/fonts` at all in the chroot (before this)

Reply:
Looking at built environments, i do see a symlink that eventually leads to /host/fonts, but you may be better versed in the FHS builder than me

Reply:
Shall I resolve this?